### DISCUSSION TITLE
I have two questions.....

### DISCUSSION THREAD

**Discussion by Luiz0047** [yokoffing/Betterfox#249] [Q&A]
First let me know if here isn't the right the right place to be asking questions, i'm not used to GitHub so i don't know how things work here ! And second, i'm loving this ! Is what i wish firefox was by dafult.
I have two general  concerns about Betterfox : Why is OCSP disabled by default and what is this "Tell websites not to sell or share my data" option ? And why it isn't avaible on vanilla firefox ? And using it wouldn't make me more unique ?

**Comment by yokoffing** [comment]
Welcome to Betterfox and Github :)

> Why is OCSP disabled by default 

OCSP is disabled in favor of CRLite; [it's faster and more private](https://blog.mozilla.org/security/tag/crlite/).

https://github.com/yokoffing/Betterfox/blob/035624935356f6b0af36202fb57cf47caabf374c/Securefox.js#L254-L263

If you want to consult OCSP for revoked certificates, then you can add the following to your overrides (the bottom of your `user.js`):
```javascript
user_pref("security.OCSP.enabled", 1); // re-enable OCSP
user_pref("security.pki.crlite_mode", 3); // consult CRLite and enforce "Not Revoked" results, but defer to OCSP for "Revoked" 
```

**Comment by yokoffing** [comment]
> what is this "Tell websites not to sell or share my data" option
https://github.com/yokoffing/Betterfox/blob/035624935356f6b0af36202fb57cf47caabf374c/Securefox.js#L198-L213

> why it isn't avaible on vanilla firefox

It's there; just not enabled by default. They had introduced it to the Nightly and Beta channels a little while ago.

> And using it wouldn't make me more unique ?

Read [this](https://github.com/arkenfox/user.js/wiki/3.3-Overrides-%5BTo-RFP-or-Not%5D#-fingerprinting).
